### PR TITLE
Eval estree expression

### PR DESCRIPTION
**Pull request by NITFALL**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/eval-estree-expression/tsconfig.json
1. types/eval-estree-expression/index.d.ts
1. types/eval-estree-expression/package.json

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 229a9f4a0ae72064a3966d9ac14d83c9b637adf2
</p>

**Review by jakebailey — CHANGES_REQUESTED** [review]
Please delete the LICENSE and README; the publisher generates these files on publish.

**Comment by NITFALL** [comment]
> Please delete the LICENSE and README; the publisher generates these files on publish.

Thanks for explaining, files removed

**Comment by jakebailey** [comment]
You're also missing tests.

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/eval-estree-expression/eval-estree-expression-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c1410b545dc08001ab1c6d632c95eee12262f38d
</p>

File: types/eval-estree-expression/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function evaluate(tree: any, context: any, options: any): any;
```
Comment (jakebailey) [inline types/eval-estree-expression/index.d.ts:1]: Is `any` really the best type for these?

In this particular function's case, are you sure it doesn't return at least a `Promise`?

File: types/eval-estree-expression/eval-estree-expression-tests.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { evaluate } from "./index";
```
Comment (sandersn) [inline types/eval-estree-expression/eval-estree-expression-tests.ts:1]: Tests should import as if they were a user of the package:
```suggestion
import { evaluate } from "eval-estree-expression";
```

**Review by sandersn — CHANGES_REQUESTED** [review]

**Pull request closed without merging**